src/css/job-posts.css

```diff
@@ -0,0 +1,299 @@
+.job-posts {
+  background:var(--COLOR_DARK_BLUE) center;
+}
+
+.job-posts__container {
+  padding-top: 95px;
+  padding-bottom: 70px;
+}
+
+.job-posts__title {
+  margin-bottom: 50px;
+  font-weight: var(--FONT_WEIGHT_700);
+  color: var(--COLOR_WHITE);
+  font-size: var(--FONT_SIZE_48);
+  letter-spacing: 0.05em;
```

Comment:
Why do you use em? Are font sizes in em too? I'd rather use px

Reply:
changed to 1px